<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Configuration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>SPI </string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Channel: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_spich"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Clock Frequency:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_fsclk"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="pushButton_spisetup">
               <property name="text">
                <string>Setup</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_spi">
               <property name="text">
                <string>SPI status: </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Bandwidth </string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Upper Bandwidth:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_fH"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Lower Bandwidth:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_fL"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>DSP Cutoff:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_fdsp"/>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_BW">
                 <property name="text">
                  <string>Set</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>RAM Registers / ADC Calibration</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="pushButton_REG_Config">
             <property name="text">
              <string>RAM REG Config.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Address: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="ReadREG">
             <property name="text">
              <string>Read Value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Show_REG">
             <property name="text">
              <string>Show All</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_ADC_Cal">
             <property name="text">
              <string>Run ADC Calibraition</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Clear_Cal">
             <property name="text">
              <string>Clear Calibraion</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Communication Testing</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>WRITE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>READ</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_test"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Plot</string>
       </attribute>
       <widget class="QCustomPlot" name="plot_ch1" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>381</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
       <widget class="QCustomPlot" name="plot_ch2" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>381</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_showdata">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>90</y>
          <width>84</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1140</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMeasurement">
    <property name="title">
     <string>Measurement</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionPause"/>
    <addaction name="actionSave"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionExit_Program"/>
   </widget>
   <addaction name="menuMeasurement"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun"/>
   <addaction name="actionPause"/>
   <addaction name="actionSave"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionExit_Program"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/run_icon.png</normaloff>:/rec/img/run_icon.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/pause_icon.png</normaloff>:/rec/img/pause_icon.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/save_icon.png</normaloff>:/rec/img/save_icon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/clear_icon.png</normaloff>:/rec/img/clear_icon.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionExit_Program">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/exit_icon.png</normaloff>:/rec/img/exit_icon.png</iconset>
   </property>
   <property name="text">
    <string>Exit Program</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
